### ISSUE TITLE
reactor/classic.d.ts contains duplicate identifier `glyph`

### ISSUE THREAD

**Issue by liam-goodacre-hpe** [sencha/extjs-reactor#110]
Currently seeing the following error with classic.d.ts:

```
ERROR in ...\node_modules\@extjs\reactor\classic.d.ts
(227320,2): error TS2300: Duplicate identifier 'glyph'.

ERROR in ...\node_modules\@extjs\reactor\classic.d.ts
(227331,2): error TS2300: Duplicate identifier 'glyph'.

ERROR in ...\node_modules\@extjs\reactor\classic.d.ts
(229406,2): error TS2300: Duplicate identifier 'glyph'.

ERROR in ...\node_modules\@extjs\reactor\classic.d.ts
(229417,2): error TS2300: Duplicate identifier 'glyph'.
```

I have checked the source and it does appear to be duplicated in two places.

TypeScript version `2.2.2`.  Let me know if you need any further information about my setup.  Thanks.

**Comment by markbrocato** [comment]
Thanks! I've published a new version 0.2.16 that should fix this.

**Comment by liam-goodacre-hpe** [comment]
Wow that was quick! Thanks 👍
